packages/react-core/src/components/Timestamp/Timestamp.tsx

```diff
@@ -2,6 +2,7 @@ import * as React from 'react';
 import styles from '@patternfly/react-styles/css/components/Timestamp/timestamp';
 import { css } from '@patternfly/react-styles';
 import { Tooltip } from '../Tooltip';
+import { isValidDate } from '../CalendarMonth';
```

Comment:
I wonder if it's worth moving this helper function out of the CalendarMonth component into a more centralized helper functions file